Dico 2.2 uses an old Libtool (2.2.7a) that did not sort the output
of 'find', thereby leading to non-deterministic file name ordering
in the arguments passed to 'ar rcu' for libdico.a & co.

--- dico-2.2/build-aux/ltmain.sh	1970-01-01 01:00:00.000000000 +0100
+++ dico-2.2/build-aux/ltmain.sh	2015-11-25 09:39:30.826169050 +0100
@@ -2926,7 +2926,7 @@ func_extract_archives ()
         func_extract_an_archive "$my_xdir" "$my_xabs"
 	;;
       esac
-      my_oldobjs="$my_oldobjs "`find $my_xdir -name \*.$objext -print -o -name \*.lo -print | $NL2SP`
+      my_oldobjs="$my_oldobjs "`find $my_xdir -name \*.$objext -print -o -name \*.lo -print | sort | $NL2SP`
     done
 
     func_extract_archives_result="$my_oldobjs"
